### DISCUSSION TITLE
Azure DevOps Subscription Placement

### DISCUSSION THREAD

**Discussion by mthemark** [Azure/Enterprise-Scale#1483] [Q&A] [answered]
We have a subscription only containing Azure DevOps organization, for billing invoice reasons.

Which be a better placement option?
1. Corp
2. Platform
3. Other (open to suggestions)
 
![image](https://github.com/Azure/Enterprise-Scale/assets/67937071/4d7c929a-83de-4f58-8716-6e2b9bb832c9)

**Comment by chrisking81** [comment]
Hi @mthemark 

I work with a customer in Financial Services, and we recently had a very similar requirements. They have identified a number of use-cases, where a subscription is required **solely** for commercial/billing purposes.

We've made the decision to add a new Management Group at the same level as Platform, LZ, Sandbox for Billing Only purposes.

We've assigned a policy which restricts deploying any Azure resources to the MG, and a PIM request will need to be approved to the subscription, when linking to a service such as ADO, or making a purchase via the Marketplace etc.

A decision was made to create a separate subscription under the new MG for each requirement, as they may have different teams approving the purchase, and for simplifying the cross-charge capabilities.

This might be overkill for other organisations, but this gives them a very good separation between workloads and billing only options. 

Just an idea, and interested to see what others are doing

**Reply by mthemark** [reply]
Thanks @chrisking81,

I was thinking along the same lines also, as I don't think there will be much policies that will affect or directly involve the "DevOps organization".

As we are using ALZ-Bicep to manage subscription placement, currently this subscription sits outside "Our Company" in the "Tenant Root Group", which seems a bit weird.

I think the Billing aspect is not really relevant to the question as main reason for having the subscription solely including DevOps organization is to include that subscription in an "Invoice Section" inside "Our Company"s "Billing Profile". i.e. Company Development Cost

**Reply by jtracey93** [reply, marked as answer]
Hey both @chrisking81 & @mthemark,

Great question and conversation here.

Whilst nothing wrong with you approach @chrisking81 at all, I think I'd follow the [ALZ Tailoring Guidance](https://aka.ms/alz/tailoring) and make this another child MG of the Landing Zones MG.

As theres nothing but positives from inheriting most of the policies from the LZ MG and above and then assigning some more specific ones on top of that 👍
